helm/aladdin-demo/templates/aladdin-demo.deploy.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Chart.Name }}
+  labels:
+    name: {{ .Chart.Name }}
+spec:
+  replicas: {{.Values.deploy.replicas}}
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        # this deploymentconfig label is used by the service to find the right deploy file
+        deploymentconfig: {{ .Chart.Name }}
+    spec:
+      # Number of seconds for the containers to perform a graceful shutdown,
+      # after which it is voilently terminated. This defaults to 30s, most apps may not need to change it
+      terminationGracePeriodSeconds: {{ .Values.deploy.terminationGracePeriod }}
+      containers:
+############# aladdin-demo app container #################
+      # falcon app container that runs with uwsgi server
+      - name: {{ .Chart.Name }}
+        # Docker image for this container
+        image: {{ .Chart.Name }}:local
```

Comment:
{{ .Values.deploy.ecr }}{{ .Chart.Name }}:{{ .Values.deploy.imageTag }} would be best